fab/inventory/icds

```diff
@@ -72,28 +109,35 @@ kafka0
 
 [celery:children]
 celery0
+celery1
 
 [pillowtop:children]
 pil0
 
 [touchforms:children]
-pil0
+pil1
```

Comment:
Should we take any special care when moving touchforms over while doing ansible deploy?